src/flux/task-plan.coffee

```diff
@@ -459,6 +463,8 @@ TaskPlanConfig =
 
     isStatsFailed: (id) -> !! @_stats[id]
 
+    hasChanged: (id) -> not _.isEqual(@exports.getChanged.call(@, id), @_local[id].defaultPlan)
```

Comment:
Hmm, should we switch `isChanged`? I'm not sure what the difference is between `.hasChanged(id)` and `.isChanged(id)` is.